pkg/provision/providers/qemu/launch.go

```diff
@@ -94,7 +83,7 @@ type LaunchConfig struct {
 	// filled by CNI invocation
 	tapName string
 	vmMAC   string
-	ns      ns.NetNS
+	nsPath  string
```

Comment:
this seems not to be used now :thinking:

Reply:
It's still used, but the logic has just moved to a function called `startQemuCmd`, which is implemented separately on darwin and linux. The linux implementation still runs the command in the new qemu network namespace.
https://github.com/siderolabs/talos/blob/665774cda8d8dbc82a2f53041504af563fb8b6b9/pkg/provision/providers/qemu/launch.go#L329-L331
https://github.com/siderolabs/talos/blob/665774cda8d8dbc82a2f53041504af563fb8b6b9/pkg/provision/providers/qemu/launch_linux.go#L218-L226

Maybe with the current name `startQemuCmd` it's unclear that on linux the command is started in a different ns. Perhaps it'd be more intuitive if I renamed it to `startQemuCmdOnNetwork` or something like that.